ServerCore/Pages/Teams/Create.cshtml.cs

```diff
@@ -89,6 +89,18 @@ public async Task<IActionResult> OnPostAsync()
                     return NotFound("Registration is full. No further teams may be created at the present time.");
                 }
 
+                //Isolate team name and Event ID
+                string teamName = Team.Name;
```

Comment:
It looks like these three variables aren't being used. Can you please remove them?